packages/ubuntu_desktop_installer/test/configure_secure_boot/configure_secure_boot_page_test.dart

```diff
@@ -64,10 +62,12 @@ void main() {
     expect(find.byType(SuccessIcon), findsNWidgets(2));
 
     await tester.enterText(fields.first, 'ubuntu');
-    verify(model.setSecurityKey('ubuntu')).called(1);
+    verify(model.setSecurityKey(ValidatedString('ubuntu', isValid: true)))
+        .called(1);
 
     await tester.enterText(fields.last, 'ubuntu');
-    verify(model.setConfirmKey('ubuntu')).called(1);
+    verify(model.setConfirmKey(ValidatedString('ubuntu', isValid: false)))
```

Comment:
Why is the expectation that the confirmation key isn't valid here?

Reply:
It was a mismatch against the mocked value from the model, but very confusing indeed because a matching value was entered above. I've made it clear by entering "mismatch" instead :)